### DISCUSSION TITLE
Not getting address information on orders for few sellers

### DISCUSSION THREAD

**Discussion by vmishra-godaddy** [etsy/open-api#1327] [Q&A]
Hello There,

We are seeing complaints from our mutual customers that address information does not exist on orders and on checking the API response, I found that etsy is not sending them
<img width="260" alt="Screenshot 2024-11-25 at 10 02 32 AM" src="https://github.com/user-attachments/assets/d194b962-6979-4550-ab34-5d5f49caddee">

Can someone please help me understand why are we not receiving address information? We are developer partner with commercial access.

**Comment by namlv94** [comment]
etsy recently removed this information for some users. This is causing problems for many apps. Please refer to: https://help.etsy.com/hc/en-us/articles/26654295371031-How-to-Use-a-Third-Party-Provider-to-Ship-Your-Order?campaign_label&segment=selling

**Reply by vmishra-godaddy** [reply]
Thanks @namlv94 but in that link it is mentioned that if it was connected before October 21, 2024, services will continue to function as usual.

> If you're a US seller and you've integrated a third-party shipping service other than Shippo or ShipStation before October 21, 2024, these services will continue to function as usual, with shipping data being automatically imported and exported.

**Reply by namlv94** [reply]
Yes. Users of your app who connected before October 21, 2024 will still be able to get that information. But users who connect after that date will be restricted. We are currently experiencing the same issue with our users.

**Reply by rcosta5** [reply]
> Thanks @namlv94 but in that link it is mentioned that if it was connected before October 21, 2024, services will continue to function as usual.
> 
> > If you're a US seller and you've integrated a third-party shipping service other than Shippo or ShipStation before October 21, 2024, these services will continue to function as usual, with shipping data being automatically imported and exported.


Did you end up finding a workaround or solution for this?

**Reply by SrdjanPaunovic** [reply]
My team and I sent a few emails but never received an update regarding this problem. Etsy support is so slow and useless.
We also requested to whitelist our app, but they just ignore everything we sent.

**Comment by swiftmyday** [comment]
So basically forcing people to scrape the website.  This is beyond ridiculous.

**Reply by mggevaer** [reply]
Which unfortunately they explicitly say you are not allowed to do, and requires full access by the user to their account rather than limited access. It is hard to understand what they are trying to achieve with this.

**Reply by CreateWithPrint** [reply]
My workaround was to use make.com to push the address info into a local database so I could match it with etsy api on my internal crm

**Reply by rrllc-3d** [reply]
> My workaround was to use make.com to push the address info into a local database so I could match it with etsy api on my internal crm

I am trying to get address to ship an order.  Doesn't make.com use your Etsy personal api access?  Don't you run into the same issue where the address information is nulled out?

**Reply by CreateWithPrint** [reply]
> > My workaround was to use make.com to push the address info into a local database so I could match it with etsy api on my internal crm
> 
> I am trying to get address to ship an order. Doesn't make.com use your Etsy personal api access? Don't you run into the same issue where the address information is nulled out?

Sorry just now seeing this.  I auth through Make's integration with etsy.  Another workaround is connecting your store to Shipstation and retrieving the address that way.  You get the customer email as well
